identity/id.go

```diff
@@ -10,8 +11,11 @@ import (
 	"github.com/mr-tron/base58"
 )
 
+// IDLength contains the amount of bytes of the ID
+const IDLength = sha256.Size
```

Comment:
Please note that changing this will also require changing the GoShimmer code that uses `sha256.Size` to refer to the length of the identity

Reply:
Ah yes. I'll revert to `sha256.Size` then.